CHANGELOG.md

```diff
@@ -94,6 +99,7 @@
 * Playlists: move tracks with Alt + Up/Down/PageUp/PageDown/Home/End
   [#13092](https://github.com/mixxxdj/mixxx/pull/13092)
   [#10826](https://github.com/mixxxdj/mixxx/issues/10826)
+  [#13098](https://github.com/mixxxdj/mixxx/pull/13098)
```

Comment:
This point is listed twice now

Reply:
I cannot confirm. Where is the second occurrence?

Reply:
![grafik](https://github.com/user-attachments/assets/b348cf3c-e468-4dae-93cd-57413f3f9d28)